src/main/java/com/google/step/finscholar/servlets/JsonConverter.java

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.step.finscholar.servlets;
+
+import com.google.gson.Gson;
+import com.google.step.finscholar.data.College;
+import java.util.List;
+
+/** This class hold all of the methods convertToJson() for all of our data objects. */
+public class JsonConverter {
+
+  /**
+   * Converts an List<College> to a json string.
+   * @param listOfColleges - List<College> that needs to be converted.
+   * @return - Newly converted JSON string.
+   */
+  public static String convertToJson(List<College> listOfColleges) {
```

Comment:
Maybe we can use polymorphism here?
We can do `convertToJson(List<T extends Serializable> listOfObject` so that other class can use this helper class?

Reply:
Just changed it to a single line!